src/shared/actions/events/event.js

```diff
@@ -42,7 +42,7 @@ export const fetchEvent = (fetch) => (
     const { id, token } = location ? location.query : {};
 
     if (id && token) {
-      return fetch(`${apiEndpoint}/event?id=${id}&token=${token}`)
+      return fetch(`${apiEndpoint}/event/${id}?token=${token}`)
```

Comment:
@lpil - not thus far
@sebflipper - this may cause issues with our preview logic as it expects `id` as a query param.